hangman.py

```diff
@@ -83,10 +76,49 @@ def get_available_letters(letters_guessed):
     returns: string (of letters), comprised of letters that represents which letters have not
       yet been guessed.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-    
-    
+    alphabet = string.ascii_lowercase
+    new_alphabet = []
+    for letter in alphabet:
+        if letter in letters_guessed:
+            new_alphabet += ''
+        else:
+            new_alphabet += f'{letter}'
+
+    return ''.join(new_alphabet)
+
+
+def victory(guesses, secret_word):
+    """
+    guesses: count of guesses left
+    secret_word: string, the word the user is guessing
+    returns: string with congratulations and total score
+    """
+    total_score = guesses * len(set(secret_word))
+    return f'\033[7mCongratulations! You won, the secret word was - {secret_word}\033[0m\n\033[7mYour total score is {total_score}\033[0m'
+
+
+def letter_check(guesses, warnings):  # currently unstable
+    """
+    guesses: count of guesses left
+    warnings: count of warnings left
+    returns: confirmed letter
+    """
+    while 1:
+        letter = input('Enter your guess letter:   ')
+        if letter.isalpha():
+            break
+        else:
+            if warnings <= 0:
+                guesses -= 1
+                if guesses == 0:
+                    break
+            else:
+                warnings -= 1
+            print(
+                f'\033[31mYou must enter a letter of the Latin alphabet. Not a number or symbol. {warnings} warnings left\033[0m')
+        print(f'WARNINGS: {warnings}, GUESSES: {guesses}')
+    return letter
+
 
 def hangman(secret_word):
```

Comment:
This function doesn't contain any comments or empty lines (to split logical parts). Please - add it